packages/eas-cli/src/commands/update/view.ts

```diff
@@ -1,11 +1,30 @@
 import { Command, flags } from '@oclif/command';
 import Table from 'cli-table3';
 import gql from 'graphql-tag';
+import groupBy from 'lodash/groupBy';
```

Comment:
Yea, I remember that, but since we are already using it else where it seemed fine to use.

Will look into reducing dependency on lodash in a seperate PR.